core-bundle/src/Resources/contao/elements/ContentMarkdown.php

```diff
@@ -19,6 +19,13 @@
  */
 class ContentMarkdown extends ContentElement
 {
+	public function __construct($objElement, $strColumn = 'main')
+	{
+		trigger_deprecation('contao/core-bundle', '4.12', 'ContentMarkdown has been deprecated and will be removed in Contao 5.0.');
```

Comment:
We usually just place this inside the class file, as soon as the class is loaded the deprecation is triggered. No need for the constructor method.